src/Microsoft.AspNetCore.Mvc.Razor/DependencyInjection/MvcRazorMvcBuilderExtensions.cs

```diff
@@ -42,6 +45,28 @@ public static class MvcRazorMvcBuilderExtensions
         }
 
         /// <summary>
+        /// Registers tag helpers as services and changes the existing <see cref="ITagHelperActivator"/>
+        /// for an <see cref="ServiceBasedTagHelperActivator"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/> instance this method extends.</param>
+        /// <returns>The <see cref="IMvcBuilder"/> instance this method extends.</returns>
+        public static IMvcBuilder AddTagHelpersAsServices(this IMvcBuilder builder)
```

Comment:
O_o. We definitely want this bit to be consistent. If we need to add new pieces we should ensure that they're doing the same thing.